anomaly-benchmarking/generate-datasets/metadata.json

```diff
@@ -11,6 +11,30 @@
       "description": "The resource ids of the 'mother' datasets"
     },
     {
+      "name": "diff-map",
+      "type": "map",
+      "default": {"easy": 0.13333333, "medium": 0.33333333, "hard": 0.5, "very hard": 1},
+      "description": "How to categorize a row's point difficulty. This is a map of category levels (e.g. \"medium\") to the maximum allowed value for that level. The default is {\"easy\" 0.13333333 \"medium\" 0.33333333 \"hard\" 0.5 \"very hard\" 1}."
+    },
+    {
+      "name": "var-map",
+      "type": "map",
+      "default":{"high scatter": 0.25, "medium scatter": 0.5, "low scatter": 1, "low clusteredness": 2, "medium clusteredness": 4, "high clusteredness": 1000000},
+      "description": "How to categorize a dataset's semantic variance. This is a map of category levels (e.g. \"low clusteredness\") to the maximum allowed ratio of variance of normal points to variance of anomalous points. The default is {\"high scatter\" 0.25 \"medium scatter\" 0.5 \"low scatter\" 1 \"low clusteredness\" 2 \"medium clusteredness\" 4 \"high clusteredness\" 1000000}."
+    },
```

Comment:
the `default` attribute in an `input` is used to set the value used as default in the execution, it can be a map, but I think that it has to ve a valid JSON object, no?
On the other hand, the "description" of the input is displayed in a tooltip if it is too long and this ones still looks too long even for the tooltip. I would recommend to write such that long definitions in the general description of the script. Charles did something like that for the SMACDown script.